src/Spectre.libGaussianMixtureModelling/ExpectationMaximization.h

```diff
@@ -86,17 +82,18 @@ namespace Spectre::libGaussianMixtureModelling
         /// </returns>
         GaussianMixtureModel ExpectationMaximization::EstimateGmm()
         {
+            constexpr DataType minLikelihoodChange = 0.00000001;
```

Comment:
[Coding convention](https://github.com/spectre-team/spectre/wiki/Cpp-coding-convention): this qualifies as *constant*, right?